src/api/uploads/MultiputUpload.js

```diff
@@ -496,13 +495,6 @@ class MultiputUpload extends BaseMultiput {
 
```

Comment:
Noticed on L488 a reference to DigestComputing, is that still relevant?

Reply:
Good catch, will remove